packages/service/tests/utilities/measurements/test_measurement/__init__.py

```diff
@@ -0,0 +1,95 @@
+"""Contains utility functions to test loopback measurement service. """
+
+import pathlib
+from typing import Iterable, Tuple
+
+import ni_measurement_plugin_sdk_service as nims
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types import xydata_pb2
+
+
+service_directory = pathlib.Path(__file__).resolve().parent
+measurement_service = nims.MeasurementService(
+    service_config_path=service_directory / "TestMeasurement.serviceconfig",
+    version="0.1.0.0",
+    ui_file_paths=[
+        service_directory,
+    ],
+)
+
+
+@measurement_service.register_measurement
+@measurement_service.configuration("Float In", nims.DataType.Float, 0.06)
+@measurement_service.configuration("Double Array In", nims.DataType.DoubleArray1D, [0.1, 0.2, 0.3])
+@measurement_service.configuration("Bool In", nims.DataType.Boolean, False)
+@measurement_service.configuration("String In", nims.DataType.String, "sample string")
+@measurement_service.configuration(
+    "String Array In", nims.DataType.StringArray1D, ["String1", "String2"]
+)
+@measurement_service.configuration("Path In", nims.DataType.Path, "path/test")
+@measurement_service.configuration(
+    "Path Array In", nims.DataType.PathArray1D, ["path/test1", "path/ntest2"]
+)
+@measurement_service.configuration("IO In", nims.DataType.IOResource, "resource")
+@measurement_service.configuration("Pin Array In", nims.DataType.PinArray1D, ["pin1", "pin2"])
+@measurement_service.configuration("Integer In", nims.DataType.Int32, 10)
+@measurement_service.output("Float out", nims.DataType.Float)
+@measurement_service.output("Double Array out", nims.DataType.DoubleArray1D)
+@measurement_service.output("Bool out", nims.DataType.Boolean)
+@measurement_service.output("String out", nims.DataType.String)
+@measurement_service.output("String Array out", nims.DataType.StringArray1D)
+@measurement_service.output("Path Out", nims.DataType.Path)
+@measurement_service.output("Path Array Out", nims.DataType.PathArray1D)
+@measurement_service.output("IO Out", nims.DataType.IOResource)
+@measurement_service.output("Pin Array Out", nims.DataType.PinArray1D)
+@measurement_service.output("Integer Out", nims.DataType.Int32)
+@measurement_service.output("XY Data Out", nims.DataType.DoubleXYData)
+def measure(
+    float_input: float,
+    double_array_input: Iterable[float],
+    bool_input: bool,
+    string_input: str,
+    string_array_input: Iterable[str],
+    path_input: str,
+    path_array_input: str,
```

Comment:
Shouldn't these type-hints be `Path` and `List[Path]`?

Reply:
Referred other measurements and I used `str`. Updated to `Path`.